src/org/omegat/gui/main/MainWindowMenu.java

```diff
@@ -291,11 +291,13 @@ public void menuCanceled(MenuEvent e) {
             }
         });
         projectExitMenuItem = createMenuItem("TF_MENU_FILE_QUIT");
+        projectRestartMenuItem = createMenuItem("TF_MENU_FILE_RESTART");
 
         // all except MacOSX
         if (!Platform.isMacOSX()) {
             projectMenu.addSeparator();
             projectMenu.add(projectExitMenuItem);
+            projectMenu.add(projectRestartMenuItem);
         }
```

Comment:
The menu item doesn't appear to be available anywhere on macOS. I think it should go *somewhere*. Maybe at the bottom of the Project menu.

Also I'm not sure about putting it below Exit; I think it's probably strongly expected that Exit is at the very bottom. What about Restart and then Exit?

So macOS would have

- (other Project menu content)
- divider
- Restart

Other platforms would have the above *and* then:

- Exit

```suggestion
        projectMenu.add(projectRestartMenuItem);
        projectMenu.addSeparator();

        // all except MacOSX
        if (!Platform.isMacOSX()) {
            projectMenu.add(projectExitMenuItem);
        }
```